app/presenters/curation_concerns/generic_work_show_presenter.rb

```diff
@@ -78,5 +78,11 @@ def show_thumb_member_presenters
         end
       end
     end
+
+    def riiif_file_id
+      return unless member_presenters.present?
+      representative_presenter = member_presenters.select{ |pres| pres.id == representative_id }.pop
+      representative_presenter.riiif_file_id
+    end
```

Comment:
Hmm. I guess we gotta think through what's happening. 

If all the member_presenters are already loaded, what's here is fine. Do we think they always/usually will be?

If they are not, and we have enough information to just create the desired presetner, it would _probably_ be a lot more performant to just do that, than to create _all_ the member presenters (which might not be needed in this action).  Since I think creating all of the member presenters may require _multiple_ solr requests, one for each?

Hard to know for sure, because I don't completely understand what's going on at the lower levels with fetches from where when. 

What do you think?